src/main.ts

```diff
@@ -11,7 +11,7 @@ import config from './config'
 import * as Utils from './utils'
 import { DeepPartial, Config, FileExtension } from './types'
 
-const RLS_VERSION = "1.1.0"
+const RLS_VERSION = "1.2.4"
```

Comment:
In fact, it's not used but I left it just to indicate current RLS version somehow. Not sure if it's a good idea as correctness is not enforced by anything.

Reply:
We can put the rls_version to package.json and add a simple built script via [pre-commit](https://www.npmjs.com/package/pre-commit) to check and download new rls binaries before each commit, this can make things easier.

Reply:
Sounds good! I can implement it during the next bump 👍

Reply:
I just get some time so I implemented on branch [rls-update](https://github.com/reasonml-editor/atom-ide-reason/commit/0a43c8f61b2c64045e21cd940c0c671b7bb3a360), looks reason-language-server@v1.2.4 is not tagged yet, so I cannot use this to bump 1.2.4. 

Your PR has just published to ide-reason@v1.1.1, thanks!

Reply:
Awesome, thanks!